examples/js/blockchaincom-withdrawal.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const ccxt = require('../../ccxt.js')
+const asTable = require('as-table')
+const log = require('ololog').configure({ locate: false })
+
+
+const exchange = new ccxt.blockchaincom({
+    'secret': 'SECRET',
+})
+
+// blockchaincom specific internal beneficiary id
+const withdrawal_beneficiary = 'WITHDRAWAL_BENEFICIARY'
+
+
+
+require('ansicolor').nice
```

Comment:
@DoctorSlimm thx! Would be great if you could clean out the requirement for `ansicolor` from it.

Reply:
Sure thing